pkgs/by-name/nl/nlohmann_json_schema_validator/package.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, nlohmann_json
+, cmake
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "nlohmann_json_schema_validator";
+  version = "2.3.0";
+
```

Comment:
```suggestion

  outputs = [ "out" "dev" ];

```

Reply:
I don't even understand this yet and how to learn how to know we have to do that :)